ssp

```diff
@@ -666,6 +668,7 @@ sub dnsonly_checks_only {
     print_kernelcare_info();
     print_cpanel_info();
     check_for_cpanel_update();
+    print_time();
     print_uptime();
     print_logged_in_users();
```

Comment:
I know this came in as part of the previous pull request, but, this output is going to be large on some servers and most of the time knowing the names of all the users who are logged in isn't going to be useful.  This also added an external binary execution and I would rather keep 'uptime' in its place which reported only the number of logged in users, among other things.

Reply:
I was indeed concerned by the risk of adding such a thing;  I wished that dave had not decided to merge the PR -- I wanted feedback particularly about what to do in the situation there may be many logged in users.  I would assume limiting the output would make the most sense, as is done in other functions.

As to just replacing one shell out with another, I agree we aren't really gaining anything.  The only way this makes sense is if in the future we parallelize more of these checks; in which case the added granularity would prove beneficial.

Reply:
I won't be signing off on anything further until @cPanelChrisD signs off.

Reply:
OK, I think I'm gonna revert the fiddling with uptime altogether.

Reply:
done